app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx

```diff
@@ -20,56 +20,59 @@ import classNames from 'classnames/bind';
 import { connect } from 'react-redux';
 import { defineMessages, injectIntl } from 'react-intl';
 import { reduxForm, formValueSelector } from 'redux-form';
+import Parser from 'html-react-parser';
 import DOMPurify from 'dompurify';
+import { Checkbox, Modal } from '@reportportal/ui-kit';
 import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
-import { DEFAULT_PROJECT_ROLE, ROLES_MAP } from 'common/constants/projectRoles';
+import { EDITOR, VIEWER } from 'common/constants/projectRoles';
 import { URLS } from 'common/urls';
 import { fetch } from 'common/utils/fetch';
 import { commonValidators } from 'common/utils/validation';
 import { urlProjectSlugSelector } from 'controllers/pages';
 import { isAdminSelector } from 'controllers/user';
 import { showScreenLockAction, hideScreenLockAction } from 'controllers/screenLock';
 import { showNotification, NOTIFICATION_TYPES } from 'controllers/notification';
-import { withModal, ModalLayout, ModalField } from 'components/main/modal';
+import { withModal, ModalField } from 'components/main/modal';
 import { FieldProvider } from 'components/fields/fieldProvider';
 import { FieldErrorHint } from 'components/fields/fieldErrorHint';
-import { showModalAction } from 'controllers/modal';
+import { hideModalAction, showModalAction } from 'controllers/modal';
 import { areUserSuggestionsAllowedSelector } from 'controllers/appInfo';
 import { AsyncAutocomplete } from 'components/inputs/autocompletes/asyncAutocomplete';
-import { InputDropdown } from 'components/inputs/inputDropdown';
 import { MEMBERS_PAGE_EVENTS } from 'components/main/analytics/events';
-import { projectKeySelector } from 'controllers/project';
+import { projectKeySelector, projectNameSelector } from 'controllers/project';
 import { InputUserSearch, makeOptions } from 'components/inputs/inputUserSearch';
 import { Input } from 'components/inputs/input';
+import { withTooltip } from 'componentLibrary/tooltip';
```

Comment:
Can we use tooltip from ui-kit?

Reply:
the tooltip in the ui-kit is not suitable